tietokanta/src/main/resources/db/migration/V1_1111__.sql

```diff
@@ -0,0 +1,62 @@
+-- Rahavaraus id:n lisäys ja populointi -- SELECT populoi_rahavaraus_idt(); 
+-- Palauttaa päivitetyt rivit yhteenlaskettuna 
+CREATE OR REPLACE FUNCTION populoi_rahavaraus_idt()
+RETURNS INTEGER AS $$
+  DECLARE
+    rivit_paivitetty INTEGER := 0;
+  BEGIN
+    -- ~ ~ toteutuneet_kustannukset ~ ~ --
+    -- Lisää rahavaraus_id sarakkeet, on olemassa jo parissa taulussa, mutta ei haittaa 
+    ALTER TABLE kulu_kohdistus ADD COLUMN IF NOT EXISTS rahavaraus_id INT REFERENCES rahavaraus (id);
+    ALTER TABLE kustannusarvioitu_tyo ADD COLUMN IF NOT EXISTS rahavaraus_id INT REFERENCES rahavaraus (id);
+    ALTER TABLE toteutuneet_kustannukset ADD COLUMN IF NOT EXISTS rahavaraus_id INT REFERENCES rahavaraus (id);
+
+    -- Äkilliset hoitotyöt 
+    UPDATE toteutuneet_kustannukset
+       SET rahavaraus_id = ra.id
+      FROM (SELECT id FROM rahavaraus WHERE nimi LIKE '%Äkilliset hoitotyöt%' ORDER BY id ASC LIMIT 1) ra
```

Comment:
Tämän id:n voisi ottaa talteen: (SELECT id FROM rahavaraus WHERE nimi LIKE '%Äkilliset hoitotyöt%' ORDER BY id ASC LIMIT 1)
Eli laita DECKAREEN akillinen_rahavaraus_id INTEGER;
Ja heti BEGININ alkuun tuo haku näin:
akillinen_rahavaraus_id := (SELECT id FROM rahavaraus WHERE nimi LIKE '%Äkilliset hoitotyöt%' ORDER BY id ASC LIMIT 1)
Ja sitten käytä sitä tässä.
Tuossa on myös muita rahavaraus_id hakuja, jotka voisi ottaa talteen heti kärkeen.